cmd/tools/commentMonitor/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const prombenchURL = "http://prombench.prometheus.io"
```

Comment:
hm , we need to avoid having this as hard coded.
I mean if the url changes we want to avoid braking things.

Maybe add it as a flag and than will provide it as an env variable or a flag when starting this action. $DOMAIN_NAME

Reply:
Added it as an env variable in the main.workflow